go-apps/meep-virt-engine/server/virt-engine.go

```diff
@@ -175,6 +176,21 @@ func msgHandler(msg *mq.Msg, userData interface{}) {
 	case mq.MsgScenarioActivate:
 		log.Debug("RX MSG: ", mq.PrintMsg(msg))
 		activateScenario(msg.Payload[fieldSandboxName])
+	case mq.MsgScenarioUpdate:
+		log.Debug("RX MSG: ", mq.PrintMsg(msg))
+		eventType := msg.Payload[fieldEventType]
+		sandboxName := msg.Payload[fieldSandboxName]
+		nodeName := msg.Payload[fieldNodeName]
+		switch eventType {
+		case mod.EventAddNode:
+			addScenarioNode(sandboxName, nodeName)
+		case mod.EventModifyNode:
```

Comment:
Thought we would not support the Modify operation
How much testing went into modifying a running process?

Reply:
The frontend currently supports only Add & Remove of processes.
The API supports Add/Remove & Modify of physical locations & processes. 
The Virt Engine supports Add/Remove & Modify of physical locations & processes. Basic manual testing of the Modify was done for now and the Virt Engine successfully redeployed the process. If the TC Engine & MG Manager components cannot easily support these API changes during implementation then we can disable support Modify. For now I would leave it enabled.